src/modules/credentials/services/CredentialService.ts

```diff
@@ -86,6 +97,7 @@ export class CredentialService {
       connectionId: connectionRecord.id,
       state: CredentialState.ProposalSent,
       proposalMessage,
+      attachments: config?.linkedAttachments?.map((linkedAttachment) => linkedAttachment.attachment),
```

Comment:
I think attachments should be called linkedAttachments. Maybe also store the whole `LinkedAttachment` object?

```suggestion
      linkedAttachments: config?.linkedAttachments?.map((linkedAttachment) => linkedAttachment.attachment),
```

Reply:
I chose to store the attachments only and not the linkedAttachment as I felt like it is redundant and does not improve usability. And if they are not `linkedAttachments` then I would just like to call them attachments to stay consistent.